client/tests/integration/manuscript-manager-template-test.coffee

```diff
@@ -23,7 +23,10 @@ module 'Integration: Manuscript Manager Templates',
   setup: ->
     app = startApp()
     server = setupMockServer()
-
+    Tahi.__container__.lookup(
+      'controller:admin/journal/manuscript-manager-template/edit'
+    )._actions.saveTemplateOnClick = -> console.log 'No Action'
+      
```

Comment:
Good question.  I should restore the function in the teardown. 

I was running into a timing issue after adding the autosave functionality.  Once a test finishes, the session is destroyed in `teardown`.  But by navigating away from the MMT, the teardown triggers an autosave event.  Then the test fails because there are records that are `inFlight`.